docs/modules/develop/pages/classes/cells.adoc

```diff
@@ -0,0 +1,30 @@
+= Cells
+
+
+As explained in dedicated page xref:develop:view_models_aka_cells.adoc[View Models (Cells)], a cell is an object that represent a fragment of your UI
+
+In order to define a cell, you need to create a class that inherits from `Decidim::ViewModel` in your plugin.
+
+Cells classes are located in the `app/cells/decidim/<my_plugin>` directory, and named: `<my_cell>_cell.rb`.
+
+For example, if you want to create a cell that renders a simple text:
+
+```ruby
+# app/cells/decidim/my_plugin/my_cell_cell.rb
```

Comment:
According to #2396, the preferred nomenclature is using the word "module" instead of "plugin", so this should be 
```suggestion
# frozen_string_literal: true

# app/cells/decidim/my_module/my_cell_cell.rb
```

You'd need to update the other examples and namespaces too of course

Also I think we should be consistent with the filenames and the frozen string comment in all the examples.